### PR TITLE
Adding ability to specify a table from Sequelize instance to use for sessionModel table

### PR DESCRIPTION
**Pull request by ScottONeal**
It would appear, in my testing, that the default model is always being used. I have a use case where I need to specify a specific table in my Sequelize instance to use for the session.

I added the option of 'table' to the options list when creating the store

```
new SequelizeStore({
  db: db, 
  table: 'ADifferentSessionTable'
});
```

I also moved the test.js into a test directory as I added some more tests to make sure this functionality works.

### CODE REVIEW

**Comment by mweibel** [comment]
hi,

thanks for your contribution. You have some intendation errors. Could you make sure this is fixed?

**Comment by ScottONeal** [comment]
Fixed. Looks like discrepancies were caused from a difference in my vim tabstop config. Essentially, it converts tabs to \s\s. Let me know if there is anything else.

File: lib/connect-session-sequelize.js [hunk 1]
```diff
@@ -25,7 +25,17 @@ module.exports = function SequelizeSessionInit(Store) {
 		}
 		Store.call(this, options);
 
-		this.sessionModel = options.db.import(path.join(__dirname, 'model'));
+    		// Check if specific table should be used for DB connection
+    		if ( options.table ) {
```
Comment (mweibel) [inline lib/connect-session-sequelize.js]: could you also remove the spaces here within the brackes?

File: lib/connect-session-sequelize.js [hunk 2]
```diff
@@ -25,7 +25,17 @@ module.exports = function SequelizeSessionInit(Store) {
...
+      			debug('Using table: %s for sessions', options.table);
+      			// Get Specifed Table from Sequelize Object
+      			this.sessionModel =  options.db[options.table] || options.db.models[options.table];
+    		}
+    		// No Table specified, default to ./model
+    		else {
```
Comment (mweibel) [inline lib/connect-session-sequelize.js]: and here please else on the same line as the closing `}`

**Comment by mweibel** [comment]
Sorry for the nits, otherwise looks fine and will merge & release a new version after you've updated

**Comment by ScottONeal** [comment]
No problem! I removed the whitespace, assuming I correctly understood what you were referencing.

**Comment by mweibel** [comment]
Merged, released a new version: 2.2.0

**Comment by mweibel** [comment]
Thanks :beers:

**Comment by ScottONeal** [comment]
Prost! :beer:

**Pull request merged**